ambry-frontend/src/main/java/com/github/ambry/frontend/GetAccountsHandler.java

```diff
@@ -128,7 +128,13 @@ private Callback<Void> securityPostProcessRequestCallback() {
           serialized = AccountCollectionSerde.serializeContainersInJson(Collections.singletonList(container));
           restResponseChannel.setHeader(RestUtils.Headers.TARGET_ACCOUNT_ID, container.getParentAccountId());
         } else {
-          serialized = AccountCollectionSerde.serializeAccountsInJson(getAccounts());
+          boolean ignoreContainers =
+              RestUtils.getBooleanHeader(restRequest.getArgs(), RestUtils.Headers.IGNORE_CONTAINERS, false);
+          if (ignoreContainers) {
+            serialized = AccountCollectionSerde.serializeAccountsInJsonNoContainers(getAccounts());
+          } else {
+            serialized = AccountCollectionSerde.serializeAccountsInJson(getAccounts());
+          }
```

Comment:
can we merge these two methods to one method with extra parameter? Something like
```
serialized = AccountCollectionSerde.serializeAccountsInJson(getAccountsI(), RestUtils.getBooleanHeader(restRequest.getArgs(), RestUtils.Headers.IGNORE_CONTAINERS, false));
```
Basically move the boolean logic to AccountCollectionSerde.

Reply:
Sure. Done..